extras/indexing/src/main/java/org/apache/rya/indexing/entity/model/Property.java

```diff
@@ -18,21 +18,21 @@
  */
 package org.apache.rya.indexing.entity.model;
 
-import static java.util.Objects.requireNonNull;
+import edu.umd.cs.findbugs.annotations.DefaultAnnotation;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import org.apache.http.annotation.Contract;
+import org.apache.http.annotation.ThreadingBehavior;
+import org.apache.rya.api.domain.RyaIRI;
+import org.apache.rya.api.domain.RyaType;
 
 import java.util.Objects;
 
-import org.apache.http.annotation.Immutable;
-import org.apache.rya.api.domain.RyaType;
-import org.apache.rya.api.domain.RyaIRI;
-
-import edu.umd.cs.findbugs.annotations.DefaultAnnotation;
-import edu.umd.cs.findbugs.annotations.NonNull;
+import static java.util.Objects.requireNonNull;
 
 /**
  * A value that has been set for an {@link TypedEntity}.
  */
-@Immutable
+@Contract(threading = ThreadingBehavior.IMMUTABLE)
```

Comment:
Explainer: https://dev-aux.com/java/org-apache-http-annotation-threadsafe-class-not-found